src/features/game/types/bumpkinItemBuffs.ts

```diff
@@ -79,6 +79,12 @@ export const BUMPKIN_ITEM_BUFF_LABELS: Partial<Record<BumpkinItem, BuffLabel>> =
       boostTypeIcon: powerup,
       boostedItemIcon: CROP_LIFECYCLE.Parsnip.crop,
     },
+    "Sunflower Shield": {
+      shortDescription: translate("bumpkinItemBuff.sunflower.shield.boost"),
+      labelType: "success",
```

Comment:
```suggestion
      labelType: "vibrant",
```
We also use vibrant if not yield boosting